components/atoms/DropDown/DropDown.scss

```diff
@@ -0,0 +1,11 @@
+.dropDown {
```

Comment:
let's reduce the amount of custom CSS we write use tailwind [@apply directive](https://tailwindcss.com/docs/functions-and-directives)  instead